src/utils/shared.ts

```diff
@@ -195,3 +195,18 @@ export const applyClientSideFormCustomizations = (properties: any) => {
   };
   return organizedProperties;
 };
+
+export const DBFilters: Record<string, string[]> = {
```

Comment:
Funny, I was just looking at this `shared.ts` file this morning and was thinking it's getting a little long and unfocused.  I wonder if there's a more specific file we could put shared component helper functions like this?  I think having the shared function is good but in this location it's kind of far from where it's used, at least it seems like it.  Nothing necessary to change for this PR though, just me pontificating :smile:

Reply:
Maybe apart from having global `shared.ts` in `/utils`, we can also create a shared file within each folder(components, machines, pages) catering to those particular sections WDYT?

Reply:
Yeah, that sounds like a good idea!  It's at worth trying it out and see how it looks over time.